gms/impl/src/main/resources/index/data-process/settings.json

```diff
@@ -44,6 +44,13 @@
           "type": "custom",
           "tokenizer": "slash_tokenizer"
         },
+        "lowercase_keyword": {
```

Comment:
Why this change?

Reply:
Good question! The test caught something :) 

I'm not sure if it was the _right_ fix, but it was a fix. The auto complete template for data process was referencing this (my guess is someone copied it from another index, but didn't copy the settings). Elasticsearch threw an error since it didn't exist.

https://github.com/linkedin/datahub/blob/d89fa4a27bdca3406e830720f650f75da6f2bdab/gms/impl/src/main/resources/dataProcessESAutocompleteQueryTemplate.json

Reply:
(I figured this fix was fine to autocomplete on lower case sensitivity, but this can always be changed if that was incorrect)